content-services/latest/develop/extension-inspector.md

```diff
@@ -114,12 +117,13 @@ When you run the Inventory command, the output is a report in JSON format with t
 ```
 
 ## Run Analyser application
+
 The application analyzes custom extensions against WAR inventories.
 
 Use the following command to run the application:
 
 ```bash
-java -jar alfresco-extension-inspector-<version>.jar <extension-filename> [--target-version=6.1.0[-7.0.0] | --target-inventory =/path/to/war_inventory.json] [--verbose=[true | false]]
+java -jar alfresco-extension-inspector-<version>.jar <extension-filename> [--target-version=6.1.0[-7.2.0] | --target-inventory =/path/to/war_inventory.json] [--verbose=[true | false]]
 ```
 
 where:
```

Comment:
Adding following text could help to understand what `extension-filename` means:
| extension-filename | Mandatory. AMP or JAR file containing an Alfresco Extension